deploy/lib/secret-traitor-stack.ts

```diff
@@ -87,31 +88,69 @@ export class SecretTraitorStack extends cdk.Stack {
         })
 
         ///////////// ACTUAL SECRET TRAITOR RESOURCES /////////////
-        const table = new dynamodb.Table(this, 'Games', {
-            tableName: 'Games',
-            partitionKey: { name: 'PK', type: dynamodb.AttributeType.STRING },
-            sortKey: { name: 'SK', type: dynamodb.AttributeType.STRING },
-        })
-
         // API Gateway REST API with the GraphQL API
-        const graphQLLambda = new lambda.Function(this, 'GraphQLHandler', {
-            runtime: lambda.Runtime.NODEJS_12_X,
+        const HttpHandler = new lambda.Function(this, 'HttpHandler', {
+            code: lambda.Code.fromAsset('../backend/dist'),
+            environment: { NODE_ENV: 'production' },
             handler: 'lambda.httpHandler',
+            memorySize: 512,
+            runtime: lambda.Runtime.NODEJS_12_X,
+            timeout: cdk.Duration.seconds(30),
+        })
+        const httpEvent = new events.Rule(this, 'http', {})
+        const RestApi = new apigw.LambdaRestApi(this, 'RestApi', {
+            handler: HttpHandler,
+        })
+
+        const WebsocketHandler = new lambda.Function(this, 'WebSocketHandler', {
             code: lambda.Code.fromAsset('../backend/dist'),
-            environment: {
-                ENV_VAR_X: process.env.ENV_VAR_X || 'default-value-x',
-                NODE_ENV: 'production',
-            },
+            environment: { NODE_ENV: 'production' },
+            handler: 'lambda.webSocketHandler',
+            memorySize: 512,
```

Comment:
Always take at least 3008 MB of memory.

The only downside of taking more memory is that Lambdas cost more per millisecond of execution the higher the allocated memory size.

But our app will be so small, and the cost is so low anyway, that this increase in cost doesn't matter.

Moreover, because the amount of compute resources (vCPUs) scales proportionally to the allocated memorySize, the Lambdas might actually take less time to complete with a higher memorySize allocation, and possibly take so much less time that they wind up costing less.

The limit was increased last week from 3008 MB to 10,240 MB, so we could go even higher. But I know some people used to say to always just use 3008 because of the economic argument I stated above, so we could stay at 3008, since I'm not sure if the tradeoffs are still generally in favor of always increasing memory once you go above 3008.